travis_ci_flake8.cfg

```diff
@@ -0,0 +1,5 @@
+[flake8]
+# temporarily disable violations for existing code until they are cleaned up
+extend-ignore =
+    # E501: line too long
```

Comment:
This has roughly 60 violations, so I didn't want to clean them up in this pull request. Does this project generally stick to the 79 characters limit?

Reply:
Hi @winniehell, thanks for all your efforts in cleaning up the codebases.
Yes, it *should* actually stick to the 79 characters limit, but I'm not sure with the route decorators, which might exceed this limit here and there...